examples/gke/my_test.sh

```diff
@@ -19,6 +19,7 @@ set -o nounset
 set -o pipefail
 
 readonly IMAGE_REPOSITORY="myrepo/chart-testing"
+readonly IMAGE_REPOSITORY="v1.0.0"
```

Comment:
Did you mean for the second `IMAGE_REPOSITORY` to be `IMAGE_TAG`?

Reply:
Fixed.